department-of-veterans-affairs/caseflow
# Extract frontend build config

![omg indeed](https://user-images.githubusercontent.com/25331532/34691581-fe2e0ed4-f48a-11e7-81f2-239d896a8c79.png)


In this PR, we extract some content to https://github.com/department-of-veterans-affairs/appeals-frontend-toolkit/pull/1. The files that are deleted from this PR are copied, largely unchanged, into appeals-frontend-toolkit.

During the PR, we install that module from GitHub. After https://github.com/department-of-veterans-affairs/appeals-frontend-toolkit/pull/1 is approved, we'll publish it on npm, and update this PR to install from npm. Then we'll be able to merge this PR.

## Known Issues
Legacy NodeJS tests fail because we are not running `node_modules` through Babel, but `appeals-frontend-toolkit` needs to be compiled.

## To Test
1. See that the karma tests still pass.
1. See that the webpack build still works (`yarn run build`).
1. See that the apps that use `ReduxBase` work.
